### ISSUE TITLE
Build MoltenVK from command line

### ISSUE THREAD

**Issue by norman784** [KhronosGroup/MoltenVK#115]
I'm wondering if theres a way to build the project entirely from the command line, I want to automate the process and this will be very helpful. I'm a bit clueless on how to doit, but seems that running `xcodebuild` does half of the job but fails at some point, so I think theres something that I'm missing in the `xcodebuild` arguments, tried with `-configuration Debug` and set a specific architecture but even so fails.

Regards

**Comment by norman784** [comment]
Sorry, I didn't post earlier the output.

[xcodebuild.log](https://github.com/KhronosGroup/MoltenVK/files/1856544/xcodebuild.log)

**Comment by karl-lunarg** [comment]
You need to run the `External/fetchDependencies` script and then run something like

```
xcodebuild -project MoltenVKPackaging.xcodeproj \
    GCC_PREPROCESSOR_DEFINITIONS='$GCC_PREPROCESSOR_DEFINITIONS MVK_LOGGING_ENABLED=0' \
    -scheme "MoltenVK (Release)" build
```

You can leave off the `GCC_PREPROCESSOR_DEFINITIONS` part.  It is just there in this case to make sure logging is off for release builds.

**Comment by billhollings** [comment]
Thanks, @karl-lunarg. I'll add this to the MoltenVK documentation.

**Comment by norman784** [comment]
Thanks @karl-lunarg, it worked so well. I'll leave open since seems that will be added to the documentation.

**Comment by billhollings** [comment]
Fixed in PR #118.
